FileUploader: Update Customize how to (#7859)
DevExpress/devextreme-documentation: concepts/05 UI Components/FileUploader/50 Customize FileUploader.md

- `place-self: center`: Centers the button and label in the `.dx-fileuploader-content` grid container.

When users upload files, the component lists selected files in its container. To move this list below the FileUploader drop zone, specify absolute positioning and implement the following styles:

    <!-- tab: styles.css -->
    .dx-fileuploader-wrapper {
        overflow: visible;
    }

    .dx-fileuploader-show-file-list .dx-fileuploader-files-container {
        padding-top: 0;
    }

    .dx-fileuploader-files-container {
        position: absolute;
        top: calc(100% + 16px);
        left: 8px;
        right: 8px;
        width: auto;
        max-height: 180px;
        overflow-y: auto;
        box-sizing: border-box;
    }

Note the following styles: 

- `top: calc(100% + 16px)`: Positions the uploaded file list container below the FileUploader.
- `overflow-y: auto`: Adds a scroll bar to the uploaded file list when necessary.

[note] To ensure absolute positioning applies to `.dx-fileuploader-files-container`, specify relative positioning for the `.dx-fileuploader-content` selector.

> The component's default behaviour is to upload selected files immediately. To allow users to select and upload files separately, set the [uploadMode](/api-reference/10%20UI%20Components/dxFileUploader/1%20Configuration/uploadMode.md '/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadMode') property to *"useButtons"*. This creates multiple upload buttons in the FileUploader container:

Reviewer: ```suggestion
The component uploads selected files immediately by default. To allow users to select and upload files separately, set the [uploadMode](/api-reference/10%20UI%20Components/dxFileUploader/1%20Configuration/uploadMode.md '/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadMode') property to *"useButtons"*. This creates multiple upload buttons in the FileUploader container:
```

Author: I feel we can simplify this sentence from the original version while simultaneously adding a touch more info—what exactly "by default"/"default behavior" means:
```suggestion
The component uploads selected files immediately when [uploadMode](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadMode) is *"instantly"* (default). To allow users to select and upload files separately, set **uploadMode** to *"useButtons"*. This creates multiple upload buttons in the FileUploader container:
```

Reviewer: I agree with your suggestion
